siliconcompiler/flows/fpga_nextpnr_flow.py

```diff
@@ -0,0 +1,80 @@
+import siliconcompiler
```

Comment:
why not make this and fpga_vpr_flow a single flow with an input to indicate VPR vs NEXTPNR?

Reply:
nextpnr does not appear to have the ability to run placement, routing, and bitstream generation as separate steps.  It seemed clunky to have FPGA targets and flows for two different place and route tools when they wouldn't share a flowgraph or anything of substance besides their synthesis front end and the users were going to have to pass the tool selection in as a load_target() argument anyway.  That said, I could be talked in to re-unifying the flow with some additional pointers on the methodology for if-elsing the flow graph based on a tool selection.

Reply:
Flows have been consolidated.  ICE40 and ZeroASIC demo architecture are offered as distinct targets with subtly different settings.